src/service/CinemaService.java

```diff
@@ -0,0 +1,179 @@
+package service;
+
+import exception.AlreadyTakenSeatException;
+import exception.NonExistingSeatException;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class CinemaService {
+
+    private final Cinema cinema;
+
+    public CinemaService() {
+        cinema = initializeCinema();
+        createSeats();
+    }
+
+    private Cinema initializeCinema() {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter the number of rows:");
+        int rows = scanner.nextInt();
+
+        System.out.println("Enter the number of seats in each row:");
+        int cols = scanner.nextInt();
+
+        System.out.println();
+
+        return new Cinema(rows, cols);
+    }
+
+    private void createSeats() {
+        String[][] seats = new String[cinema.getRows()][];
+
+        for (int i = 0; i < cinema.getRows(); i++) {
+            String[] row = new String[cinema.getColumns()];
+            Arrays.fill(row, "S");
+            seats[i] = row;
+        }
+
+        cinema.setSeats(seats);
+    }
+
+    public void showMenu() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            System.out.print("""
+                    1. Show the seats
+                    2. Buy a ticket
+                    3. Statistics
+                    0. Exit
+                    """);
+
+            String input = scanner.nextLine();
+            System.out.println();
+
+            switch (input) {
+                case "0" -> {
+                    return;
+                }
+                case "1" -> showSeats();
+                case "2" -> buyTicket();
+                case "3" -> statistics();
+                default -> {
+                    System.out.printf("Unknown menu item - %s\n", input);
+                    showMenu();
+                }
+            }
+        }
+    }
+
+    private void showSeats() {
+        System.out.print("Cinema:\n ");
+        for (int i = 1; i <= cinema.getSeats()[0].length; i++) {
+            System.out.print(" " + i);
+        }
+
+        System.out.println();
+
+        int counter = 1;
+        for (String[] seat : cinema.getSeats()) {
+            System.out.print(counter);
+            for (String s : seat) {
+                System.out.print(" " + s);
+            }
+            System.out.println();
+            counter++;
+        }
```

Comment:
Can you add tabulation when printing the matrix?

![image](https://user-images.githubusercontent.com/27965136/145179726-2e3ca55b-5b52-4e80-8de8-4bdca3d6cb4f.png)

Reply:
Fixed.